recipes/antlr4-cppruntime/all/conandata.yml

```diff
@@ -8,7 +8,26 @@ sources:
   "4.10.1":
     url: "https://github.com/antlr/antlr4/archive/refs/tags/4.10.1.tar.gz"
     sha256: "a320568b738e42735946bebc5d9d333170e14a251c5734e8b852ad1502efa8a2"
+  "4.11":
+    url: "https://github.com/antlr/antlr4/archive/refs/tags/4.11.0.tar.gz"
+    sha256: "317f4e745badd8b5272191c3a4fd34a183a242aef269c68a9b23a382590dd142"
+  "4.11.1":
+    url: "https://github.com/antlr/antlr4/archive/refs/tags/4.11.1.tar.gz"
+    sha256: "81f87f03bb83b48da62e4fc8bfdaf447efb9fb3b7f19eb5cbc37f64e171218cf"
 patches:
   "4.9.3":
     - patch_file: "patches/0001-update-cmakelist.patch"
-      base_path: "source_subfolder"
+    - patch_file: "patches/0004-update-DropLibuuid-4.9.3-1.patch"
```

Comment:
please add the metadata  patch fields so we can track these better 🙏 

https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields

I would also suggestion opening a PR upstream with the fixes